dist/ember-table.js

```diff
@@ -16,156 +16,72 @@ var define, requireModule, require, requirejs;
   } else {
     _isArray = Array.isArray;
   }
-
-  var registry = {}, seen = {};
+  
+  var registry = {}, seen = {}, state = {};
   var FAILED = false;
 
-  var uuid = 0;
```

Comment:
@LydiaBarnes none of that should have changed with your PR. You need to rebuild the dist files.